onadata/apps/api/viewsets/xform_list_viewset.py

```diff
@@ -94,6 +94,9 @@ def filter_queryset(self, queryset):
             if self.action == 'list' and profile and xform_pk is None:
                 forms_shared_with_user = get_forms_shared_with_user(
                     profile.user)
+                if self.request.query_params.get('formID'):
+                    forms_shared_with_user = forms_shared_with_user.filter(
+                        id_string=self.request.query_params.get('formID'))
```

Comment:
Since we're extracting this twice, and it's not a trivial extraction, what do you think about pulling it out into a var above then referencing that var in these two locations, e.g.:

```python
id_string = self.request.query_params.get('formID')

if id_string:
    forms_shared_with_user = forms_shared_with_user.filter(id_string=id_string)
```